src/briefcase/console.py

```diff
@@ -176,7 +203,7 @@ def _log(
                 markup = True
             for line in message.splitlines():
                 self.print(
-                    f"{preface}{prefix}{line}",
+                    f"{escape(self._context)}{preface}{prefix}{line}",
```

Comment:
Same as other escape [comment](https://github.com/beeware/briefcase/pull/1106/files#r1118118986). (gh won't let me select all this as one block)